res/enkf/es_update.py

```diff
@@ -187,6 +187,7 @@ def smootherUpdate(self, run_context):
         updatestep = local_config.getUpdatestep()
 
         analysis_config = self.ert.analysisConfig()
+        analysis_config.set_verbose(self.ert.get_verbose())
```

Comment:
Conseptually I dont quite agree with this, as there is no (direct) way for the user to set this property on the analysis_config, it will always be overwritten by what is written on the command line. Would it not be more transparent to just forward the enkf_main verbose flag? Or come to think of it, get rid of the verbosity all together. Does it serve a purpose to print something in the terminal? We dont really want that I guess.